src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -284,7 +285,7 @@ const styles = StyleSheet.create({
     paddingLeft: normalize(20),
     justifyContent: 'space-between',
     alignItems: 'center',
-    backgroundColor: 'white'
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to my previous comment, changing the background color to `COLORS.almostBlack` could impact the visibility of other components. Please make sure all elements within this component are clearly visible against the new background color.
```diff
-    backgroundColor: 'white'
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->